tests/check_dicom.c

```diff
@@ -1,9 +1,21 @@
+#include "config.h"
+
+#ifdef HAVE_CHECK
+
+#include <stdio.h>
 #include <stdlib.h>
 #include <check.h>
 
-#include "../src/dicom.h"
+#include "dicom.h"
 
 
+static char *fixture_path(const char *relpath)
+{
+    void *path = malloc(strlen(SRCDIR) + strlen(relpath) + 2);
```

Comment:
Maybe `char *path`? sprint() to a void and them an implicit cast on return looks odd.